cocos/core/math/quat.ts

```diff
@@ -424,6 +452,7 @@ export class Quat extends ValueType {
     }
 
     /**
+     * @en Calculating the three-dimensional quaternion matrix information, default zoom information input matrix does not contain
```

Comment:
Calculates the quaternion with the three-dimensional transform matrix, considering no scale included in the matrix